packages/commercetools/theme/pages/Checkout.vue

```diff
@@ -5,7 +5,7 @@
         <SfSteps
           v-if="!isThankYou"
           :active="currentStepIndex"
-          :class="{ 'checkout__steps': true, 'checkout__steps-auth': isAuthenticated }"
+          class="checkout__steps"
```

Comment:
Nice. So `isAuthenticated` is redundant after removing PersonalDetails. I think you could get rid of `useUser` from this component in that case